.github/workflows/Build.yml

```diff
@@ -36,7 +36,11 @@ jobs:
         run: npm run update-version -- "${{ env.NIGHTLY_VERSION }}"
 
       - name: Build extension
-        run: npm run build
+        env:
+          R2_ENDPOINT: https://f5dbfc5cf299cd799bf49f258675f82b.r2.cloudflarestorage.com
+          AWS_ACCESS_KEY_ID: ${{ secrets.SOURCE_MAP_BUCKET_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.SOURCE_MAP_BUCKET_SECRET_ACCESS_KEY }}
+        run: npm run build:nightly
```

Comment:
Uhm i might be wrong here but are all builds gonna get processed by the nightly script? shouldn't we have a different script for release like `build:release`

Reply:
Yeah i have some work left to do for this pr. It doesn't actually do anything specific to nightly builds -- just needs to be renamed

Reply:
gotchu, nw! feel free to tag me again once done